p/themes/Swage/swage.scss

```diff
@@ -838,8 +838,13 @@ form {
 			margin-top: 3px;
 		}
 
-		.dropdown .icon {
-			filter: brightness(3);
+		.dropdown .dropdown-menu {
+			max-height: calc(100vh - 45px);
+			overflow: scroll;
+
+			.icon {
```

Comment:
Is this change on purpose? It looks quite unrelated to the overflow stuff.

Reply:
it was by purpose.

`.dropdown .icon` is the same like `.dropdown .dropdown-menu .icon`, so `.dropdown .dropdown-menu` can be used for the (new) scrolling and for the `.icon`